samples/nrf5340/netboot/README.rst

```diff
@@ -7,82 +7,99 @@ nRF5340: Network core bootloader
+     #. Once the copy is done, the sample compares the SHA of the data in the application partition against the SHA specified in the shared SRAM.
+     #. It then communicates the result of the comparison to the application core through the shared SRAM.
+
+#. Locking the flash memory areas containing the application.
+     The sample locks the flash memory areas that contain the application.
      Locking is done using the ACL peripheral.
      For details on locking, see the :ref:`fprotect_readme` driver.
 
-#. Boot the application on the network core.
-     After possibly performing a firmware update, and enabling flash protection, the network core bootloader uninitializes all peripherals that it used and boots the application.
+#. Booting the application on the network core.
+     After possibly performing a firmware update, and enabling flash memory protection, the network core bootloader uninitializes all peripherals that it used and boots the application.
+
+The most common use case for the network core bootloader sample is to be included as a child image in a multi-image build, rather than being built stand-alone.
 
 Requirements
 ************
 
-  * |nRF5340DK|
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpunet
 
 .. _net_bootloader_build_and_run:
 
 Building and running
 ********************
 
-The source code of the sample can be found under :file:`samples/nrf5340/netboot/` in the |NCS| folder structure.
+The recommended way of building the sample is to use the :ref:`multi-image feature <ug_multi_image>` of the build system.
 
-The most common use case for the network core bootloader sample is to be included as a child image in a multi-image build, rather than being built stand-alone.
-This sample is included automatically if the application in the nRF5340 network core has the :option:`CONFIG_SECURE_BOOT` option set.
-Note that the build system will automatically enable the :option:`CONFIG_SECURE_BOOT` option for the network core application if the application on the application core has enabled :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53`.
+You can include the sample in your multi-image build in two different ways:
 
-A new set of firmware update files is generated by the build system when this image is included in a multi-image build.
-These files are identical to those described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
+* Enabling the :option:`CONFIG_SECURE_BOOT` option in the application that runs on the nRF5340 network core.
+  With this option enabled, the build system includes the sample automatically.
+* Enabling the :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53` options in the application that runs on the application core.
```

Comment:
Users don't need to give `CONFIG_BT_RPMSG_NRF53` themselves - this is included automatically when compiling for netboot.

The only options required for the user to give in order to build this sample are `mcuboot_CONFIG_PCD=y` and `CONFIG_BOOTLOOADER_MCUBOOT=y`

Reply:
fixed